### ISSUE TITLE
Disable friendly fire from projectiles

### ISSUE THREAD

**Issue by coffenbacher** [SFTtech/openage#553]
It seems that getting in the way of your own projectiles causes friendly fire.

![image](https://cloud.githubusercontent.com/assets/245867/15094707/04e70416-1462-11e6-8655-5d4683eeb74b.png)

**Comment by TheJJ** [comment]
Mind that some projectiles must cause friendly fire (Mangonel etc). Somewhere in the data must be a flag for the friendly fire activation.

**Comment by coffenbacher** [comment]
Right, I think the distinction is that friendly fire only applies to area of effect projectiles.

Here's some research I found on the original behavior. I'll try to find some research on the "artillery" projectiles like trebuchets, mangonel line, bombard cannon, bombard tower etc. as well to get a complete picture of how the mechanics work.

[Accuracy 1 - SotL](https://www.youtube.com/watch?v=HpzOTdItUGA)
* Archers have 100% accuracy within two tiles
* "Accuracy" stat means that percentage of shots automatically target correct tile. Inaccurate shots are fired at a random vector in the same general direction, but also have a chance of hitting the correct tile if the random vector is close enough to the same direction. 
* Accuracy is a function of distance, giving credence to the vector theory. More tiles = more inaccurate.
* Thumb ring = 100% accuracy. However, will still miss moving targets without ballistics.
* Ballistics = shots are targeted at where the enemy unit will be, as opposed to where it is. Same accuracy rules apply.

[Accuracy 2, Inaccurate Arrows - SotL](https://www.youtube.com/watch?v=DZQn4-aAjGU)
* Inaccurate arrows do full damage if they happen to hit targeted unit
* Inaccurate arrows do 50% damage (i.e. calculate damage normally with armor, then divide by two) if they happen to hit a different unit

[Accuracy 3, Unit Size - SotL](https://www.youtube.com/watch?v=33MEpQKCTes)
* Inaccurate arrows more likely to hit a large unit than a small unit. I.e. at 3 tiles, an elephant is hit 96% of time vs. 83% for a samurai.

**Comment by TheJJ** [comment]
If you like, this is good information to add in the `doc/reverse_engineering` directory :)
